src/managers/WebhookManager.js

```diff
@@ -198,8 +216,10 @@ class WebhookManager {
           auth: webhook.auth,
           body: requestFormData ? null : requestJSON || event.data,
```

Comment:
This here might be incorrect.  

The body can either be `JSON` or it can be `Form Data`.  Both of these can use hogan to pass in parameters to the request.

The Form Data may come from the device as `foo={{SPARK_CORE_ID}}&bar=123` or `foo=123&bar=1234`.  
In the first case, it will need to be run through hogan.